modules/ting/ting.entities.inc

```diff
@@ -352,7 +364,17 @@ class TingEntity extends DingEntity {
    *   The description.
    */
   public function getDescription() {
-    return empty($this->reply) ? FALSE : $this->reply->getDescription();
+    $description = FALSE;
```

Comment:
Are you sure this does not belong in https://github.com/easySuite/ding2/blob/development-4.4.x/modules/opensearch/src/OpenSearchTingObject.php#L393?

Reply:
Please relate to my comment.

Reply:
In https://github.com/easySuite/ding2/blob/development-4.4.x/modules/opensearch/src/OpenSearchTingObject.php#L393 will be outputted all text found in description. Code proposed to be added will trim description value to predefined amount of symbols. Don't we need such functionality?

Reply:
No. Because in a final we are processing TingEntity object which is going through TingEntity class declared in ting/ting.entities.inc.